TSDB privileges

awhoward opened the pull request:
Attempted fix of Issue https://github.com/Keck-DataReductionPipelines/KPF-Pipeline/issues/1212

Checking CI with this PR.  Hold off on merging until I can test it.

RussLaher commented:
It is correct, assuming the table-creation step is done with the kfpadminrole.  On the other hand, if the table is created with the kpfporole, then I think it should own the table:
`ALTER TABLE {self.prefix}metadata OWNER TO kpfporole;`

awhoward commented:
@RussLaher -- I went around and around on this and decided that it makes more sense to have all of the table creation and dropping accomplished by `kpfadminrole`.  These tasks will happen infrequently and one of us and log in with appropriate privileges to do so.  Therefore, I removed the line below and a similar one.  Sound reasonable?
```
ALTER TABLE {self.prefix}metadata OWNER TO kpfporole;
```

I also updated Readthedocs with instructions for how to rebuild the tables.

This is ready to merge if Russ thinks it looks reasonable.

awhoward commented:
I think this will resolve Issue https://github.com/Keck-DataReductionPipelines/KPF-Pipeline/issues/1212

awhoward commented:
The previous commit addresses Russ' suggestion from yesterday.
